sheet_to_triples/field.py

```diff
@@ -96,7 +96,12 @@ def as_hash(self):
 
     @property
     def as_uc(self):
-        return _must(self._pattern.sub('', _str(self._value).title()))
+        return _must(
+            ''.join(
+                [x.title() if x.islower()
+                 else x for x in re.findall(r'\w+', self._value)]
+            )
+        )
```

Comment:
Do you want this change in this PR? It's not clearly related to conditionals.

I suggest removing from this proposal, and re-proposing along side some tests that are explaining what the expected outcome behaviour is, because the logic as written looks pretty slow and fragile compared to the current.